README.md

```diff
@@ -97,9 +98,9 @@ func main() {
 	n, err := minioClient.FPutObject(bucketName, objectName, filePath, minio.PutObjectOptions{ContentType:contentType})
 	if err != nil {
 		log.Fatalln(err)
+	} else {
+		log.Printf("Successfully uploaded %s of size %d\n", objectName, n)
```

Comment:
This else is not needed.

Reply:
Both ways have the same affect, but this is about the style. 
In all of our code, we do not use `else` if not needed.

Reply:
@davegallant the reason we don't worry about `else` clause here is because log.Fatalln(err) will exit the program and only upon success we get to log.Printf() . So this doesn't need to be changed. Changing the `bucket` message is correct.

Reply:
thanks. fixed it. (new to golang)

Reply:
Thanks for the PR.